test/test.js

```diff
@@ -7,79 +6,86 @@ import fs from 'node:fs/promises'
 import { randomUUID } from 'node:crypto'
 import { once } from 'node:events'
 import { getPaths } from '../lib/paths.js'
+import assert from 'node:assert'
 
 const __dirname = dirname(fileURLToPath(import.meta.url))
 const station = join(__dirname, '..', 'bin', 'station.js')
 
 // From https://lotus.filecoin.io/lotus/manage/manage-fil/
 const FIL_WALLET_ADDRESS = 'f1abjxfbp274xpdqcpuaykwkfb43omjotacm2p3za'
 
-test('FIL_WALLET_ADDRESS', async t => {
-  await t.test('require address', async t => {
+describe('FIL_WALLET_ADDRESS', () => {
+  it('fails without address', async () => {
     try {
       await execa(station)
     } catch (err) {
       return
     }
-    t.fail('should have thrown')
+    throw new Error('should have thrown')
```

Comment:
Slightly unrelated to this PR, but have you considered using [`assert.rejects`](https://nodejs.org/dist/latest-v18.x/docs/api/assert.html#assertrejectsasyncfn-error-message)?

```js
assert.rejects(execa(station))
```

Reply:
That is awesome, thank you so much for the tip!!